CMakeLists.txt

```diff
@@ -65,12 +67,19 @@ function(check_platform_supports_browse_mode RESULT)
 	if(NOT inline_result EQUAL "0")
 		# The inline script failed, so browse mode is not supported.
 		set(${RESULT} "0" PARENT_SCOPE)
+		message(STATUS "Checking if browse capable - no")
```

Comment:
I would just print a warning message here (maybe not for Windows) and drop all the other output.